utils/ChromiumDownloader.js

```diff
@@ -25,15 +25,24 @@ const removeRecursive = require('rimraf');
 const ProxyAgent = require('https-proxy-agent');
 const getProxyForUrl = require('proxy-from-env').getProxyForUrl;
 
-const DOWNLOADS_FOLDER = path.join(__dirname, '..', '.local-chromium');
+const DOWNLOADS_FOLDER = path.join(__dirname, '..', process.env.CONTENT_SHELL ? '.local-shell' : '.local-chromium');
```

Comment:
chromium downloader might be run in a different environment; the process.env.CONTENT_SHELL might not be defined there.

Can we plumb "product" as a third parameter to the downloader methods?